src/components/flowchart/index.js

```diff
@@ -418,7 +414,21 @@ export class FlowChart extends Component {
    */
   handleNodeMouseOver = (event, node) => {
     this.props.onToggleNodeHovered(node.id);
-    this.showTooltip(event, node);
+    node && this.showTooltip(event, node.fullName);
+  };
+
+  handleParamsIndicatorMouseOver = (event, node) => {
+    const nodeParameterNames = Object.keys(
+      this.props.nodeParameters[node.id] || {}
+    );
+    if (nodeParameterNames.length) {
+      const label =
+        nodeParameterNames.length > 1
+          ? `Parameters:${nodeParameterNames.length}`
+          : `Params:${nodeParameterNames[0]}`;
```

Comment:
do we need `Params` before the parameter name?  perhaps displaying only the parameter name for the single parameter is enough?  (cc. @GabrielComymQB )